priv/repo/migrations/20190420102616_create_audio.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Radiator.Repo.Migrations.CreateAudio do
+  use Ecto.Migration
```

Comment:
Maybe we should call this AudioFile to keep the door open for an Audio that is more of the meta object we want in the future. E.g. an Audio represents a podcast episode. It consists of multiple source audio files for the tracks, and multiple processed, download ready files (e.g. the results of something coming back from auphonic), maybe in multiple versions with different inserts, etc.

Reply:
Gute Idee, ist umgesetzt